src/Scripting/CSharp/CSharpScriptCompiler.cs

```diff
@@ -11,7 +11,7 @@ internal sealed class CSharpScriptCompiler : ScriptCompiler
     {
         public static readonly ScriptCompiler Instance = new CSharpScriptCompiler();
 
-        private static readonly CSharpParseOptions s_defaultOptions = new CSharpParseOptions(kind: SourceCodeKind.Script, languageVersion: LanguageVersion.Latest);
+        private static readonly CSharpParseOptions s_defaultOptions = new CSharpParseOptions(kind: SourceCodeKind.Script, languageVersion: LanguageVersion.CSharp8);
```

Comment:
>CSharp8 [](start = 155, length = 7)

Adding @jaredpar, @tmat to review version number change.

Reply:
Can't take the version change here. The script and C# compiler are in sync with respect to versions. The script compiler will move to eight at the same time Latestn moves to 8.

Reply:
Yes, I think it makes sense to allow specify language version in ScriptOptions. An interesting question is whether different submissions in the chain can use different C# language versions. I think it should be ok. Might be actually useful, if users can do something like `#langversion 8.0` in Interactive Window (as a host command) to switch to a different version to try how some snippet works in that version.

Also the VS C# Interactive Window should switch to max version like we did for EEs.

---
In reply to: [254033944](https://github.com/dotnet/roslyn/pull/33096#discussion_r254033944) [](ancestors = 254033944)